thunder/core/module.py

```diff
@@ -141,6 +143,116 @@ def load_original_state_dict(self, state_dict):
                 else:
                     raise NotImplementedError(f"don't know how to handle {full_k}")
 
+    def state_dict(self, *, destination=None, prefix="", keep_vars=False):
+        """
+        Returns the state dict of the (transformed) Thunder module.
+
+        Args:
+            destination: if given, use this mutuable mapping as the dict container.
+            prefix: a prefix for the keys.
+            keep_vars: do not detach
+
+        Note that this similar but rather more rudimentary than the original state_dict.
+        """
```

Comment:
nice addition but this wouldn't be rendered unless we update https://github.com/Lightning-AI/lightning-thunder/blob/e88966f2f35433e167f40d26e1330684b8ee3fc1/docs/source/reference/thunder.rst?plain=1#L46, looking at https://lightning-thunder--968.org.readthedocs.build/en/968/reference/thunder.html#jited-model-wrapper

Reply:
Right, thank you!